tools/build_config/build.sh

```diff
@@ -27,6 +27,7 @@ function print_package_versions() {
   echo "$(cmake --version | head -n 1)"
   echo "Matlab: ${MATLAB_ROOT}"
   echo "$(g++ --version | head -n 1)"
+  echo "$(cppcheck --version | head -n 1)"
```

Comment:
These commands don't need the `echo "${  }"`, just  `cppcheck --version | head -n 1` will achieve the same thing

Reply:
I left this for consistency with the rest of the file; the Herbert `build.sh` has had these fixes made on  a previous iteration -- I didn't want to make more changes than needed here as you're editing the same file in https://github.com/pace-neutrons/Horace/pull/210